### ISSUE TITLE
Can you add tag 0.2.3 in your repository?

### ISSUE THREAD

**Issue by vaaho** [furf/jquery-ui-touch-punch#234]
I can not automatically create component jqueryui-touch-punch for rails.

https://rails-assets.org/components/new

When I wrote package name _jqueryui-touch-punch_ and version _0.2.3_, the rails-assets responded error: _No tag found that was able to satisfy 0.2.3._ Could you help me?

**Comment by jerrylow** [comment]
Bump ~ Same issue here with our projects.

**Comment by agroendijk** [comment]
Would be great.

**Comment by zatchgordon** [comment]
I got around the issue by putting "*" (an asterix) instead of the version number

**Comment by agroendijk** [comment]
@zatchgordon it's a workaround indeed but then you can't lock the version to the version that works with your project. Looking at the activity of this repository I guess it wouldn't be a big problem though.

**Comment by jerrylow** [comment]
Agree with @agroendijk just good practice for everyone all around, but good workaround though.

**Comment by zatchgordon** [comment]
True, didn't think about version control. Only thought about initially getting the file.
